studio-docs/source/federated-graphs.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Federated graphs in Studio
+sidebar_title: Federated graphs
+---
+
+<!-- don't forget to: import { VideoFrame } from 'gatsby-theme-apollo-docs' -->
+
+A [federated graph](https://apollographql.com/docs/federation) (also known as a **supergraph** ) is a graph that’s composed of multiple individual **subgraphs**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+Each subgraph represents an underlying service or set of services that provide their own GraphQL API. Different subgraphs are often owned by different teams in an organization.
+
+Apollo Studio provides tools to help multiple teams collaborate on a federated graph and its subgraphs.
+
+<!-- video will go here -->
+
+## Viewing a field's originating subgraph
+
+The **Schema > Reference** tab in Studio displays a table of your federated schema’s types and fields. This table includes a Subgraph column:
+
+<img class="screenshot" src="./img/federated-graphs/schema-reference-type-row-with-subgraph-column.jpg" width="736" />
+
+Clicking the subgraph link for a type or field takes you to the line in the subgraph schema where it's defined:
+
+<img class="screenshot" src="./img/federated-graphs/type-definition-in-subgraph-sdl.jpg" width="700" />
+
+If you're using [the `@contact` directive](#defining-subgraph-contact-information) to specify owner contact information for your subgraphs, hovering over a subgraph name displays its contact information, enabling you to follow up with the appropriate team:
+
+<img class="screenshot" src="./img/federated-graphs/schema-reference-type-row-showing-subgraph-contact-directive-tooltip.jpg" width="736" />
+
+## Viewing subgraph SDL
+
+The **Schema > SDL** tab in Studio displays the raw SDL of your federated graph’s schema, along with the schema of each subgraph.
+
+From this tab you can:
+- View metadata about the composed graph and each subgraph (such as endpoint URL and the most recent schema registration date)
+- Identify the originating subgraph for any type or field in your API and share a link with team members
+- Search for types and fields
+- Download a copy of your subgraph’s SDL for local development
+- Filter out comments and deprecated fields for improved scanning
+
+## Subgraph contact info
+
+You can use the `@contact` directive to specify contact information for a particular subgraph, such as a name and URL. This information is visible in Studio, which helps your organization members find who to contact for assistance with a particular type or field:
+
+<img class="screenshot" src="./img/federated-graphs/subgraph-contact-info-on-sdl.jpg" width="736" />
```

Comment:
I recommend we use the "closeup" screenshot here as well

Reply:
@StephenBarlow so I chose this version because it gives you a full example of what it looks like to have the contact directive in your SDL, and what it looks like when it appears on the SDL page, which might be a helpful reference since in this article we don't show all of that together in once place. Thoughts?